app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
+  def index
+  end
+end
```

Comment:
Can you please prepare setup for https://github.com/rspec/rspec-rails which will be our testing library and create basic test for this controller? You should be able to achieve this by simply following their `README.md`.